addon/components/paper-autocomplete.js

```diff
@@ -91,7 +91,11 @@ export default PowerSelect.extend(ValidationMixin, ChildMixin, {
       let publicAPI = this.get('publicAPI');
 
       if (isNone(publicAPI.selected)) {
-        publicAPI.actions.open(event);
```

Comment:
What does this do?

Reply:
This will send an API request(mimic the input functionality) when you focus in the autocomplete field provided onFocusRefresh is set to true.